README.md

```
@@ -44,57 +54,52 @@ Verifique se há alguma variavel pendente no docker-compose com o comando
 
-Veja os detalhes do metabase+SMAE em [metabase.md](metabase.md) ou a documentação completa em https://www.metabase.com/docs/latest/embedding/introduction#signed-embedding
+Na primeira vez que subir o sistema com gerenciamento de arquivos via MinIO, será necessário subir o MinIO, criar um bucket, e configurar um usuário e senha para os uploads, e então atualizar o `.env` com as configurações realizadas. Todas essas tarefas podem ser feitas pelo console web, https://min.io/docs/minio/linux/administration/minio-console.html
 
-## Restaurando um backup do banco de dados
+## 6. Restaurando um Backup (Se Aplicável)
 
 Para restaurar um backup do banco de dados PostgreSQL, siga os passos abaixo:
 
 1. **Inicie apenas o serviço do banco de dados:**
 
+```bash
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)